### ISSUE TITLE
CS231n 计算机视觉

### ISSUE THREAD

**Issue by junxnone** [junxnone/csc#16]
## Reference

- [斯坦福CS231n李飞飞计算机视觉](https://study.163.com/course/introduction.htm?courseId=1003223001#/courseDetail?tab=1)
- [CS231n: Convolutional Neural Networks for Visual Recognition](http://vision.stanford.edu/teaching/cs231n/index.html)
- [2017CS231n 斯坦福李飞飞视觉识别](https://study.163.com/course/introduction.htm?courseId=1004697005#/courseDetail?tab=1)
- [Schedule and Syllabus](http://cs231n.stanford.edu/syllabus.html)

## Lecture
- [ ] 章节1深度学习与计算机视觉
- [x] 课时1课件下载方法
- [x] 课时2计算机视觉历史回顾与介绍（上）27:29
- [ ] 课时3计算机视觉历史回顾与介绍（中）21:28
- [ ] 课时4计算机视觉历史回顾与介绍（下）25:46
- [ ] 课时5数据驱动的图像分类方式：K最近邻与线性分类器（上）29:35
- [ ] 课时6数据驱动的图像分类方式：K最近邻与线性分类器（下）28:57
- [ ] 课时7 线性分类器损失函数与最优化（上）36:30
- [ ] 课时8线性分类器损失函数与最优化（下）35:50
- [ ] 课时9反向传播与神经网络初步（上）40:59
- [ ] 课时10反向传播与神经网络初步（下）40:42
- [ ] 课时11神经网络训练细节part1（上）40:29
- [ ] 课时12神经网络训练细节part1（下）40:06
- [ ] 课时13神经网络训练细节part2（上）35:42
- [ ] 课时14神经网络训练细节part2（下）34:58
- [ ] 课时15卷积神经网络详解（上）40:12
- [ ] 课时16卷积神经网络详解（下）39:48
- [ ] 课时17迁移学习之物体定位与检测（上）37:05
- [ ] 课时18迁移学习之物体定位与检测（下）28:58
- [ ] 课时19卷积神经网络的可视化与进一步理解（上）39:48
- [ ] 课时20卷积神经网络的可视化与进一步理解（下）39:41
- [ ] 课时21循环神经网络（上）30:45
- [ ] 课时22循环神经网络（下）40:05
- [ ] 课时23卷积神经网络工程实践技巧与注意点（上）36:48
- [ ] 课时24卷积神经网络工程实践技巧与注意点（下）39:07
- [ ] 课时25深度学习开源库使用介绍（上）42:40
- [ ] 课时26深度学习开源库使用介绍（下）39:44
- [ ] 课时27图像分割与注意力模型（上）33:56
- [ ] 课时28图像分割与注意力模型（下）38:08
- [ ] 课时29视频检测与无监督学习（上）36:40
- [ ] 课时30视频检测与无监督学习（下）43:19
- [ ] 课时31来自谷歌大脑负责人Jeff Dean的受邀讲座（上）36:57
- [ ] 课时32来自谷歌大脑负责人Jeff Dean的受邀讲座（下）39:03
- [ ] 章节2配套作业与答案（中文版）
- [ ] 课时33Asignment 1-Q1 k-Nearest Neighbor (kNN) exercise
- [ ] 课时34Assignment 1-Q2 Multiclass Support Vector Machine
- [ ] 课时35Assignment 1-Q3 Softmax exercise
- [ ] 课时36Assignment 1-Q4 Two-Layer Neural Network
- [ ] 课时37Assignment 1-Q5 Image features exercises
- [ ] 课时38Assignment 2 Q1-Q3一段关于神经网络的故事
- [ ] 课时39Assignment 2  Q4 - ConvNet on CIFAR-10
- [ ] 课时40Assignment 2 Q5 - TensorFlow
- [ ] 课时41Assignment 3 Q1 - Q2
- [ ] 课时42Assignment 3 Q3
- [ ] 课时43Assignment 3 Q4
- [ ] 课时44Assignment 3 Q5
